conf/configuration.go

```diff
@@ -356,9 +357,30 @@ func updateConfigFromClowder(c *ConfigStruct) error {
 		fmt.Println(noStorage)
 	}
 
+	// get in-memory DB configuration from clowder
+	if !c.Redis.UseClowder {
```

Comment:
I guess this is needed in order to support both the cache-writer and the velkey-writer being deployed on clowder at the same time, right? I'm asking because we are not using a custom `use_clowder` config for postgres for example